web/satellite/src/components/account/billing/BillingArea.vue

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
nit - this change is unnecessary (in fact if this file was created in 2019, the copyright header should stay as 2019)

Reply:
We're getting conflicting instructions on these Copyright comments. We have previously been instructed to change them to 2022

Reply:
if it is a file you added, it should be 2022. If it already existed before your changes, the previous date is fine. See https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated/2391555#2391555 for more context

Reply:
updated to 2019